src/sso/pages/Login.tsx

```diff
@@ -94,42 +99,30 @@ function Login() {
   async function handleRequests() {
     try {
       const verifiedJwt = await UserApps.onRedirectLogin();
-
-      const tonomyJwt = (await ExternalUser.loginWithTonomy(
-        { callbackPath: "/callback", redirect: false },
-        new JsKeyManager()
-      )) as string;
-
-      sendRequestToMobile([verifiedJwt.jwt, tonomyJwt]);
+      const keymanager = new JsKeyManager();
+      const user = await ExternalUser.getUser(keymanager);
+
+      if (user) {
+        //TODO: send to the connect screen
+        alert("send to connect page");
```

Comment:
i think here we need to send to the logged in page... I don't know what the connect page is. Better make this clearer.